old-testsuite/Testsuite/Vector/AsList.hs

```diff
@@ -18,6 +18,12 @@ instance Modelled (Vector a) where
   model = V.toList
   unmodel = V.fromList
 
+naiveTakeEnd :: Int -> [a] -> [a]
```

Comment:
This test function is duplicated to be used in two files, I would rather put it in a separate module to avoid duplication, like the existing `Utilities`module.

Reply:
Moved to Utilities